src/api/routers/participantUsersRouter.ts

```diff
@@ -9,7 +10,7 @@ const createParticipantUsersRouter = () => {
   const participantUsersRouter = express.Router({ mergeParams: true });
   const userController = new UserController(new UserService(), new LoggerService());
 
-  participantUsersRouter.use('/:userId', verifyAndEnrichUser);
+  participantUsersRouter.use('/:userId', verifyAndEnrichUser, isAdminOrUid2SupportCheck);
```

Comment:
Why does `isAdminOrUid2SupportCheck` need to go here? Wouldn't this line need to be hit if the user is Operations?

Reply:
All the routes in this file require `Admin` or `UID2 Support` - this is what this line is enforcing. Perhaps it is not clear enough - I guess we want to avoid the dev having to pay attention to the order of lines here. I have just updated it so that each endpoint explicitly has the middleware check.